### PR TITLE
Galxy Energy NFTs Collectibles

### PR DESCRIPTION
**Pull request by NFTsGalaxy**
Galaxy Energy a unique Collectibles Project will be only 100 exclusive units each with their specularity where each art extracts energy from the galaxy in a different way from the other

### CODE REVIEW

**Comment by NFTsGalaxy** [comment]
Can anyone help me I can't solve this error I appreciate it

File: Galaxy Energy [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  
+
+
+
+
+          
+
+
+
```
Comment (rsegeda) [inline Galaxy Energy:60]:
```suggestion
[
   {
      "project":"Galaxy Energy",
      "tags":[
         "Collectable Arts Galaxy Energy"
      ],
      "policies":[
         "bde52528724eb3761a921dc68aa85e4fa259d5a5b50b86442229ae5f",
         "15ed06333898d4a679c5e98890341c59905b3a566c235b5bcec1ce42",
         "993c26aff20f5207b89da6790878cc640e0dbbdea0d4ff18302e3541",
         "298d1451df0a91fc6d154cd9c7dd48d01f202316edb5956891496795"
      ]
   }
]
```

**Comment by rsegeda** [comment]
> Can anyone help me I can't solve this error I appreciate it

https://github.com/Cardano-NFTs/policyIDs/pull/6209#discussion_r800220015

**Comment by KaiNFT** [comment]
Dear Contributor, 

This request will now be closed, make a new ticket with the following criteria; the new file must be within the 'Projects' directory. Please check the README.md file for further information.

**Pull request closed without merging**